thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/pojo/RawAnomalyResultBean.java

```diff
@@ -27,6 +27,24 @@
   private Long creationTimeUtc;
   private boolean dataMissing;
   private boolean merged;
+  private double currentVal;
+  private double baselineVal;
```

Comment:
Add these two fields: currentVal and baselineVal, to hashCode, equals, and compareTo methods.

You might want to use the same names on RawAnomalyResultBean and MergedAnomalyResultBean.